README.md

```diff
@@ -13,3 +13,24 @@
   complexity suggests a different time.
 
 Add your answers to this markdown file.
+
+Part 1:
+1. Asymptotic follows a formula that is a generalization, but depending on the code and how it is configured, the runtime can be much longer.
+2. Certain data may be bigger than others, so if you are running a greater amount of data on one data structure and a less amount of data on the same structure, the performance can differ.
```

Comment:
Also don't know what you mean here -- the input size is part of the asymptotic expression.